src/components/view-pane/index.js

```diff
@@ -39,28 +39,28 @@ class ViewPane extends Component {
     );
     return tagContentHeader;
   }
-  renderValues (content, index, onButtonClick) {
-    const deleteButton = <Button className='redButton' onHandleClick={onButtonClick} symbol={'x'} />;
-    const addButton = <Button onHandleClick={onButtonClick} symbol={'+'} />;
+  renderValues (content, metaIndex, onDeleteValue) {
     if (content.values) {
-      return content.values.map(value => {
+      return content.values.map((value, index) => {
+        const deleteButton = <Button className='redButton' onHandleClick={() => onDeleteValue(metaIndex, index)} symbol={'x'} />;
         return (
             <div key={value} className='inputGroup'>
                 <Input type='text' className='form-control' value={value} buttonAfter={deleteButton}/>
             </div>
           );
       });
     } else {
+      const addButton = <Button onHandleClick={onDeleteValue} symbol={'+'} />;
       return (
-        <div key={index} className='inputGroup'>
+        <div key={metaIndex} className='inputGroup'>
             <Input type='text' className='form-control' buttonAfter={addButton}/>
         </div>
       );
     }
   }
-  renderMetadataContent (item, onButtonClick, height) {
+  renderMetadataContent (item, onDeleteValue, height) {
     if (item.metadata) {
-      const addButton = <Button onHandleClick={onButtonClick} symbol={'+'} />;
+      const addButton = <Button onHandleClick={onDeleteValue} symbol={'+'} />;
```

Comment:
So just to check, these are placeholders for onAddValue which is an action to be added?